javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -374,7 +377,7 @@ public final void write(ClientMethod clientMethod, JavaType typeBlock) {
                 typeBlock.publicMethod(clientMethod.getDeclaration(), function -> {
                     AddOptionalVariables(function, clientMethod, restAPIMethod.getParameters(), settings);
                     if (clientMethod.getReturnValue().getType() == ClassType.InputStream) {
-                        function.line("Iterator<ByteBufferBackedInputStream> iterator = %s(%S).map(ByteBufferBackedInputStream::new).toStream().iterator()",
+                        function.line("Iterator<ByteBufferBackedInputStream> iterator = %s(%S).map(ByteBufferBackedInputStream::new).toStream().iterator();",
```

Comment:
bug fix.

`Formatter.formatSourceAndFixImports` will fail on invalid java code. It then fails the generate script, but not triggering error.